govc/test/namespace.bats

```diff
@@ -113,4 +113,140 @@ load test_helper
     run govc namespace.service.info -json service2
     assert_matches DE-ACTIVATED
 
+}
+
+@test "namespace.create" {
+    vcsim_env
+
+    run govc namespace.create -supervisor=domain-c1 -namespace=test-namespace-1
+    assert_success
+
+    ns=$(govc namespace.info test-namespace-1 | jq)
+    assert_equal "domain-c1" $(echo $ns | jq -r '."cluster"')
+    assert_equal "0" $(echo $ns | jq -r '."vm_service_spec"."content_libraries"' | jq length)
+
+    run govc namespace.create -supervisor=domain-c1 -namespace=test-namespace-2 -content-libraries=lib1,lib2
+    assert_success
+
+    ns=$(govc namespace.info test-namespace-2 | jq)
+    assert_equal "2" $(echo $ns | jq -r '."vm_service_spec"."content_libraries"' | jq length)
+}
+
+@test "namespace.change" {
+    vcsim_env
+
+    govc namespace.create -supervisor=domain-c1 -namespace=test-namespace-1
+
+    run govc namespace.change -content-libraries=lib1,lib2 -vm-classes=class1 test-namespace-1
+    assert_success
+
+    ns=$(govc namespace.info test-namespace-1 | jq)
+    assert_equal "2" $(echo $ns | jq -r '."vm_service_spec"."content_libraries"' | jq length)
+    assert_equal "lib1" $(echo $ns | jq -r '."vm_service_spec"."content_libraries"[0]')
+    assert_equal "lib2" $(echo $ns | jq -r '."vm_service_spec"."content_libraries"[1]')
```

Comment:
I don't think we assume the libs will be returned in this order. Would be better to test that both libs are there, but not in a specific order

Reply:
I replaced these equality checks with pattern matchers